bin/dogescript.js

```diff
@@ -5,7 +5,7 @@ var path     = require('path');
 var util     = require('util');
 var stream   = require('stream');
 var repl     = require('repl');
-var argv     = require('optimist').usage('Usage: dogescript <file>').argv;
+var argv     = require('optimist').usage('Usage: dogescript <file> [options]').argv;
```

Comment:
Options traditionally go before the file, as far as I can tell.

Also, I don't know where/if this line shows up, I just get the REPL every time I try

Reply:
i think if you don't type anything you just get the repl... i can probably add a `-h` in a following pull

Reply:
>Options traditionally go before the file, as far as I can tell. 

I think that is the case, however the repl already was defined to have the options like that... so i just kept the order and updated the doc.